Lib/fnmatch.py

```diff
@@ -61,6 +61,22 @@ def filter(names, pat):
                 result.append(name)
     return result
 
+def filterfalse(names, pat):
```

Comment:
Alternative would be to introduce an argument for negatives, was this rejected as not a good idea? Most of code would be reused.

Alternatively could factor out private function which is called from each of these.